azure/table_azure_monitor_activity_log_event.go

```diff
@@ -0,0 +1,566 @@
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ResourceProviderName.Value"),
+			},
+			{
+				Name:        "resource_type",
+				Description: "The resource type.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ResourceType.Value"),
+			},
+			{
+				Name:        "operation_name",
+				Description: "The operation name.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("OperationName.Value"),
+			},
+			{
+				Name:        "status",
+				Description: "A string describing the status of the operation. Some typical values are: Started, In progress, Succeeded, Failed, Resolved.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Status.Value"),
+			},
+			{
+				Name:        "sub_status",
+				Description: "The event sub status. Most of the time, when included, this captures the HTTP status code of the REST call. Common values are: OK (HTTP Status Code: 200), Created (HTTP Status Code: 201), Accepted (HTTP Status Code: 202), No Content (HTTP Status Code: 204), Bad Request(HTTP Status Code: 400), Not Found (HTTP Status Code: 404), Conflict (HTTP Status Code: 409), Internal Server Error (HTTP Status Code: 500), Service Unavailable (HTTP Status Code:503), Gateway Timeout (HTTP Status Code: 504)",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SubStatus.Value").Transform(transform.ToString),
+			},
+
+			// JSON fields
+			{
+				Name:        "authorization_info", // Since "authorization" is a reserved keyword in PostgreSQL, we had to rename it and use an alternative column name.
+				Description: "The sender authorization information.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Authorization"),
+			},
+			{
+				Name:        "claims",
+				Description: "Key value pairs to identify ARM permissions.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "http_request",
+				Description: "The HTTP request info. Usually includes the 'clientRequestId', 'clientIpAddress' (IP address of the user who initiated the event) and 'method' (HTTP method e.g. PUT).",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("HTTPRequest"),
+			},
+			{
+				Name:        "properties",
+				Description: "The set of <Key, Value> pairs (usually a Dictionary<String, String>) that includes details about the event.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("EventName.Value"),
+			},
+
+			// Azure standard columns
+			{
+				Name:        "resource_group",
+				Description: ColumnDescriptionResourceGroup,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ResourceGroupName"),
+			},
+		}),
+	}
+}
```

Comment:
We have missed out the `tenant_id` column please add it.

Reply:
The column `subscription_id` is available for this table. It has been moved to the common columns for all the tables. Do we add `tenant_id` in the tables?

Reply:
@ParthaI not sure if we add `tenant_id` in all tables however we do have a few tables that have `tenant_id` as a column.

Reply:
@madhushreeray30, I do not think it is required for this table.